FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +19,14 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] result in
+			if self == nil { return }
+			switch result {
+			case let .success((data, response)):
+				completion(FeedImagesMapper.map(data, response))
+			case .failure(_):
```

Comment:
Since this portion of code ignores the error from the `.failure` case, we can ignore it completely:
```suggestion
			case .failure:
```